src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (num) => {
+    if (typeof num !== 'number') {
+      const finalSum = sum;
+
+      sum = 0;
```

Comment:
do u really need this?

Reply:
i think so...
When calling without parameters, it will return the result and clear the amount.